Gateway Bucket Notifications (#10729)
Minio-io/minio: docs/bucket/notifications/README.md

# MinIO Bucket Notification Guide [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)

> Events occurring on objects in a bucket can be monitored using bucket event notifications. If you are planning to use this capability, please be aware that when used as a  Gateway (except NAS gateway) you  will not be able to use bucket notifications.

Reviewer: ```suggestion
Events occurring on objects in a bucket can be monitored using bucket event notifications. 

NOTE: gateway mode does not support bucket notifications (except NAS gateway).
```

> Event types supported by MinIO server are

Reviewer: This line explicitly mentions that it is only supported by `MinIO server`.

Author: Any chance that we can be more explicit? I think it does not hurt and it might save some time for potential users in the future.
